CONTRIBUTING.md

```diff
@@ -1,16 +1,21 @@
 # CONTRIBUTION GUIDELINES
+======
```

Comment:
I don't think we need this one. The GitHub markdown automatically adds one on the top heading (though it is not in the spec - afaik). If you look at the Contributing page, it is present. 

In fact, from what I can see, there is an actual issue on your fork [page](https://github.com/AFMahmuda/Go/blob/docs/CONTRIBUTING.md)

Reply:
adjusted